### PR TITLE
rose edit: show full traceback on macro crash

### LINKED ISSUE #1709: rose edit: show full traceback of macro crash
When a macro crashes, the exception dialog that pops up shows the string content of the exception.
It would be useful to either show the full traceback by default or have a button to show it.

### PR DESCRIPTION
**Pull request by oliver-sanders**
closes #1709 
@benfitzpatrick Please review

### CODE REVIEW

File: lib/python/rose/config_editor/menu.py [hunk 1]
```diff
@@ -697,7 +698,8 @@ def run_custom_macro(self, config_name=None, module_name=None,
             except Exception as e:
```
Comment (benfitzpatrick) [inline lib/python/rose/config_editor/menu.py]: No need for `as e` any more (not using it).

**Comment by benfitzpatrick** [comment]
One comment, otherwise looks good and tests fine.

**Comment by oliver-sanders** [comment]
@benfitzpatrick Removed nu-necessary 'as e' statement.

**Comment by benfitzpatrick** [comment]
OK, looks good to me.

**Pull request merged**